Symfony/CS/Finder.php

```diff
@@ -0,0 +1,70 @@
+<?php
```

Comment:
This file was renamed from `Symfony/CS/Finder/DefaultFinder.php`, then:
- change namespace
- stop searching for `*.xml` and `*.yml` as 2.x Finder does not do that, DefaultFinder will still search for them to not break BC